cf-agent/verify_files.c

```diff
@@ -608,9 +608,12 @@ static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp,
     Attributes a = *attr; // TODO: Try to remove this copy
     PromiseResult result = PROMISE_RESULT_NOOP;
 
+    JsonElement *destroy_this = NULL;
```

Comment:
Cool name

Reply:
Yes. The ownership of these Attributes structs is really bad, but requires a lot more effort to fix.